vnext/package.json

```diff
@@ -52,13 +52,13 @@
     "jscodeshift": "^0.6.2",
     "just-scripts": "^0.24.2",
     "prettier": "1.17.0",
-    "react-native": "https://github.com/microsoft/react-native/archive/v0.60.0-microsoft.35.tar.gz",
+    "react-native": "^0.60.0",
```

Comment:
>^0.60.0 [](start = 21, length = 7)

It seems dangerous to use the ^ in the specifier.  First a silly question - wouldn't the ^ bring in 0.62.X when it gets published?  If so that would be bad.  I thought ~ was the safer wildcard.  Second, wouldn't it be safer to snap to an explicit version, given we are an out-of-tree platform?  I worry even patch versions could be breaking changes given react-native is still not at 1.0

Reply:
^ is the same as ~ for 0.x versions.

The version is going to be locked by the lock file.  All making it a ^ really does is make dependabot kick off a PR trying to bump to a newer version if a new 0.60.x is published.  Which I think is fine, since that is something we'd want to immediately jump on, as any break there would prevent users of RNW from updating their RN version to the new version.  -- Also if there are any breaks there, it would cause react-native init apps to start failing.

Reply:
Oh yeah, I think I was looking for tilde here instead of carett, but yeah, I agree with the sentiment. yarn update could change patch version if someone runs it, which isn't ideal.

Reply:
@acoates-ms re dependabot: I was thinking that might be nice to have, but we're so far behind right now I'm not sure there will be patch versions we will want to adopt. Someone accidentally running `yarn update` scares me a bit, since Github won't show the lockfile and we've seen some breaking changes that came at runtime that CI didn't catch.